libpod/container.go

```diff
@@ -515,3 +515,23 @@ func (c *Container) CGroupPath() cgroups.Path {
 func (c *Container) StopTimeout() uint {
 	return c.config.StopTimeout
 }
+
+// RootFsSize returns the root FS size of the container
+func (c *Container) RootFsSize() (int64, error) {
+	c.lock.Lock()
```

Comment:
Can you wrap this and the one below in `if !c.locked` to speed up Batch()

Reply:
Should I wrap just the locking or the syncing as well?

Reply:
Both, please

Reply:
that is what I thought, just confirming. Fixed.